virtualization/remote_kvm.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from kvm import KVM
+
+
+class RemoteKVM(KVM):
+    name = "remote_kvm"
+    description = "Access remote KVM machines."
+
+    config = [
+        {
+            'name': 'connection_string_remote',
+            'type': 'string',
+            'description': 'KVM connection string for remote libvirtd.',
+            'default': 'qemu+ssh:///user@host/system'
+        },
+        {
+            'name': 'ssh_private_key',
+            'type': 'text',
+            'description': 'The SSH private which is to be used by KVM.',
+            'default': None
+        },
+        {
+            'name': 'ssh_private_key_path',
+            'type': 'string',
+            'description': 'Path to SSH private key file.',
+            'default': '/home/fame/.ssh/id_rsa'
+        }
```

Comment:
* Without looking at the code, I did not understand these configuration options. It is unclear that the module will write the content of `ssh_private_key` to `ssh_private_key_path`. I would probably change the description of the latter to reflect that.
* I don't think that there should be a default for `ssh_private_key_path`. Since the module does not take care of SSH authentication itself, it should be clear that the location used should be the location used by the SSH configuration to find the private SSH key.

=> Since FAME does not take care of the authentication itself, I would remove both parameters and create a README.md that explains how to setup the connection at the OS level. I guess we wouldn't need a remote_kvm module then ?

Reply:
You are right, I agree on both of your points. Nevertheless I don't follow your conclusion that we don't need the remote_kvm module then. I would rather suggest to merge both KVM modules because it is currently not possible to change the connection string of the underlying libvirt connection. That was also the main cause for creating an additional remote_kvm module.

Reply:
Fair enough. This could be a remote_kvm module or an update to the existing kvm to add a connection_string configuration option.

Reply:
I will modify the kvm module to have an option for the connection string. The default value will be the current static value to not break anything. I will also add some (short) documentation on how to setup the connection on system level.